src/app/stop-watch/stop-watch.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="watch__container">
```

Comment:
looks like BEM, but not exactly. Generally in Angular we have incapsulated styles, so no need to use BEM here, thus we can use simpler class names